Update the c2c source doc (#1454)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md

  </tr>
</table>

### Restart Sources

> Sources to can be restarted when they are in ThirdPartyConfig errors.

Reviewer: ```suggestion
You can restart the Sources when they are in ThirdPartyConfig errors.
```

> You can restart the Sources when they are in ThirdPartyConfig errors.

Reviewer: ```suggestion
When Sources encounter ThirdPartyConfig errors, they can now be restarted. Both the Sumo Logic UI and Sumo Logic API allow you to restart the Sources.
```

#### UI

> To restart your source in the Sumo Logic platform:

Reviewer: ```suggestion
To restart your source in the Sumo Logic platform, follow the steps below:
```

> 1. Open the Collection page, go to Manage Data > Collection > Collection.

Reviewer 2: ```suggestion
1. Open the Collection page, go to **Manage Data > Collection > Collection**.
```

> 2. Find the source and click the information icon on the right of the row.

Reviewer: ```suggestion
2. Select the source and click the **information** icon on the right side of the row.
```

> 3. The API usage information popup is displayed. Click the Restart Source button on the bottom left. <br>![restart source button.png](/img/collector/restart-source-button.png)</br>

Reviewer 2: ```suggestion
3. The API usage information popup is displayed. Click the **Restart Source** button on the bottom left. <br>![restart source button.png](/img/collector/restart-source-button.png)</br>
```

4. Click **Confirm** to send the restart request. <br>![restart source confirm.png](/img/collector/restart-source-confirm.png)</br>
5. The bottom left of the platform will provide a notification informing you the request was successful.<br/>   ![source restart initiated.png](/img/collector/source-restart-initiated.png)

#### API

> To restart your source via Sumo Management API

Reviewer 2: ```suggestion
To restart your source using the Sumo Management API
```

> To restart your source using the Sumo Management API

Reviewer: ```suggestion
To restart your source using the Sumo Management API, follow the method below:
```
